<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3695" started-at="2018-08-02T18:43:30Z" finished-at="2018-08-02T18:43:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3695" started-at="2018-08-02T18:43:30Z" finished-at="2018-08-02T18:43:34Z">
      <class name="com.qtpselenium.hybrid.suitea.LoginTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="init" is-config="true" duration-ms="1498" started-at="2018-08-03T00:13:30Z" finished-at="2018-08-03T00:13:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="initTest" is-config="true" duration-ms="456" started-at="2018-08-03T00:13:32Z" finished-at="2018-08-03T00:13:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="FAIL" signature="logintest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="logintest" duration-ms="286" started-at="2018-08-03T00:13:32Z" data-provider="getData" finished-at="2018-08-03T00:13:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=U1, ExpectedResult=LoginSuccess, Password=P1, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.qtpselenium.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:52)
	at com.qtpselenium.hybrid.suitea.LoginTest.logintest(LoginTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.IllegalStateException: The driver executable does not exist: D:\Common\drivers\geckodriver.exe
	at com.google.common.base.Preconditions.checkState(Preconditions.java:534)
	at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:136)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:131)
	at org.openqa.selenium.firefox.GeckoDriverService.access$100(GeckoDriverService.java:41)
	at org.openqa.selenium.firefox.GeckoDriverService$Builder.findDefaultExecutable(GeckoDriverService.java:141)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:339)
	at org.openqa.selenium.firefox.FirefoxDriver.toExecutor(FirefoxDriver.java:158)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
	at com.qtpselenium.keywords.GenericKeywords.openBrowser(GenericKeywords.java:83)
	... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="quit" is-config="true" duration-ms="859" started-at="2018-08-03T00:13:32Z" finished-at="2018-08-03T00:13:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="initTest" is-config="true" duration-ms="1" started-at="2018-08-03T00:13:33Z" finished-at="2018-08-03T00:13:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="FAIL" signature="logintest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="logintest" duration-ms="37" started-at="2018-08-03T00:13:33Z" data-provider="getData" finished-at="2018-08-03T00:13:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=itsthaku.r@gmail.com, ExpectedResult=LoginSuccess, Password=Ashish@123, Runmode=Y, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.qtpselenium.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:52)
	at com.qtpselenium.hybrid.suitea.LoginTest.logintest(LoginTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property; for more information, see https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver. The latest version can be downloaded from http://chromedriver.storage.googleapis.com/index.html
	at com.google.common.base.Preconditions.checkState(Preconditions.java:754)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:124)
	at org.openqa.selenium.chrome.ChromeDriverService.access$000(ChromeDriverService.java:32)
	at org.openqa.selenium.chrome.ChromeDriverService$Builder.findDefaultExecutable(ChromeDriverService.java:137)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:339)
	at org.openqa.selenium.chrome.ChromeDriverService.createDefaultService(ChromeDriverService.java:88)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)
	at com.qtpselenium.keywords.GenericKeywords.openBrowser(GenericKeywords.java:86)
	... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="quit" is-config="true" duration-ms="248" started-at="2018-08-03T00:13:33Z" finished-at="2018-08-03T00:13:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="logintest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="logintest" duration-ms="5" started-at="2018-08-03T00:13:34Z" data-provider="getData" finished-at="2018-08-03T00:13:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=U3, ExpectedResult=LoginFailure, Password=P3, Runmode=N, Browser=Edge}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to NO
	at com.qtpselenium.hybrid.suitea.LoginTest.logintest(LoginTest.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="initTest" is-config="true" duration-ms="0" started-at="2018-08-03T00:13:34Z" finished-at="2018-08-03T00:13:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.suitea.LoginTest@d8355a8]" name="quit" is-config="true" duration-ms="183" started-at="2018-08-03T00:13:34Z" finished-at="2018-08-03T00:13:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qtpselenium.hybrid.suitea.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
